el/index.js

```diff
@@ -38,6 +38,13 @@ module.exports = yeoman.generators.Base.extend({
         'ex: yo polymer:el my-element'
       ));
     }
+    
+    var isInsideProjectFolder = this.fs.exists('app/elements/elements.html');
```

Comment:
I think you may want this to be `this.fs.exists(this.destinationPath('app/elements/elements.html'))`